requirements/system/server.tex

```diff
@@ -1,8 +1,55 @@
 \subsection{Central Server}
+\subsubsection{Definitions}
 \begin{enumerate}
- \item CSs must be accessible through a CLI.
- \item CSs must store data on queued/running/past jobs in a database.
- \item CS job database must not change on CS crash/restart.
- \item Users must be able to query the state of jobs and work machines from a CSs via a web interface or API.
+  \item[NCS10] A job in the context of the central server consists of: (TODO: define here or in WMC??)
+  \begin{itemize}
+    \item A program to be executed.
+    \item A unix user who owns the job.
+    \item Priority of the task.(TODO: define priorities in user client and link them here, Urgent, High, Normal, Low)
+    \item Number of CPU threads needed for the execution of the program.
+    \item Amount of RAM needed for the execution of the program.
+    \item Paths in the network storage needed for the application.
+    \item One of:
+    \begin{itemize}
+      \item A Dockerfile providing instructions to build program environment.
+      \item The name of predefined container environment (TODO: link to list of environments we want to provide, in the WMC)
+    \end{itemize}
+  \end{itemize}
+  \item[NCS20] The status of a job can be: (TODO: this may be better fit for work-machine-client requirements??)
```

Comment:
I think it is better in the definitions.

Reply:
I'll move it to the glossary as @M1keReck suggested.